readme.md

```diff
@@ -79,6 +81,23 @@ Default: [`electron.session.defaultSession`](https://electronjs.org/docs/api/ses
 
 The [partition](https://electronjs.org/docs/api/session#sessionfrompartitionpartition-options) the protocol should be installed to, if you're not using Electron's default partition.
 
```

Comment:
Line 42:

```
### loadUrl = serve(options)

Reply:
What's the change needed here? The searchParameters option goes on the returned function which I added in a new section at line 91. Unless I'm missing something...